envi/archs/amd64/opcode64.py

```diff
@@ -1336,8 +1336,8 @@
 tbl32_F30FC7_rest = list( tbl32_0FC7_rest )
 
 
-desc_0FC7_00BF   = (tbl32_0FC7_00BF, 3,3, 0x07, 0, 0xbf, TBL_0FC7_rest)
-desc_0FC7_rest   = (tbl32_0FC7_rest, 3, 0, 0x07, 0xc0, 0xff)
+desc_0FC7_00BF   = (tbl32_0FC7_00BF, 3, 3, 0x07, 0, 0xbf, TBL_0FC7_rest)
+desc_0FC7_rest   = (tbl32_0FC7_rest, 3, 3, 0x07, 0xc0, 0xff)
```

Comment:
good catches!

Reply:
This should actually clear up an issue I saw in the libstdc++ stuff (since that's like the one place I saw these used).

Reply:
oh please oh please oh please oh please oh please oh please oh please oh please oh please oh please oh please oh please oh please